### PR TITLE
Add doc samples in js files

### PR DESCRIPTION
**Pull request by yohannroussel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,8 +102,6 @@ The _warped_ function can read it, and also modify the object. Modifications mad
 flow are applied and visible in the main flow.
 
 ``` javascript
-const warp = require('warp');
-
 const warp = require('warp');
```
Comment (jpl-scale) [inline README.md]: A blank line after 105

File: README.md [hunk 2]
```diff
@@ -47,7 +47,9 @@ function primeDecomposition(n, done) {
 }
 
 warp.call(primeDecomposition, 9007199254740991,
-    (a) => console.log(`Value: ${a}`));
+    (a) => {
```
Comment (jpl-scale) [inline README.md]: Not convinced

File: samples/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (jpl-scale) [inline samples/package.json]: Need to think

Comment (jpl-scale) [reply samples/package.json]: - Remove `keywords`, `repository` and `homepage`
- Add `"private":  true`
- Set version to `1.0.0`

File: README.md (hunk 1 above)
Comment (yohannroussel) [reply README.md]: Done.

File: samples/package.json (hunk 3 above)
Comment (yohannroussel) [reply samples/package.json]: Done.

File: README.md (hunk 2 above)
Comment (yohannroussel) [reply README.md]: OK reverted.

**Pull request merged**